src/components/Chip/KeywordChip.jsx

```diff
@@ -1,9 +1,18 @@
+import CloseIcon from "../Icon/CloseIcon";
+import Button from "../UI/Button";
 import PropTypes from "prop-types";
 
-const KeywordChip = ({ keywordName }) => {
+const KeywordChip = ({ keywordName, hasCloseButton, handleCloseIconClick }) => {
   return (
-    <span className="flex-center text-14 px-5 py-2 bg-orange-100 text-rose-800 rounded-[3px]">
+    <span className="flex-center text-14 px-3 py-2 bg-orange-200 rounded-[3px]">
       {keywordName}
+      <div>
+        {hasCloseButton && (
+          <Button styles="right-17" onClick={handleCloseIconClick}>
+            <CloseIcon className="size-25" />
+          </Button>
+        )}
+      </div>
     </span>
```

Comment:
`div`태그로 감싸져있어서 중앙정렬이 안 되는 것 같아요 😥 UI적으로 주의해서 보다보니 말씀드려봅니다!

아래와 같이 해주셔도 좋을 것 같아요! 

```suggestion
    <span className="flex-center gap-2 text-14 px-2 py-2 bg-orange-200 rounded-[5px]">
      {keywordName}
      {hasCloseButton && (
          <CloseIcon className="size-12" onClick={handleCloseIconClick} />
      )}
    </span>
```

저도 해당 Chip에 대해서 keywordName을 받는게 좋을 것 같다 생각하고 있었는데, 먼저 해주셨군요 :)

Reply:
오 그러네요. 바로 자식 태그가 아니라 flex 적용이 안되겠네요. 수정하도록 할게요 !